src/pretix/presale/templates/pretixpresale/event/index.html

```diff
@@ -296,6 +296,7 @@ <h5>
                                                         <div class="input-group input-group-price">
                                                             <span class="input-group-addon">{{ event.currency }}</span>
                                                             <input type="number" class="form-control input-item-price"
+                                                                    {% if ev.presale_has_ended %}disabled{% endif %}
```

Comment:
Could you check for ``not ev.presale_is_running`` instead? Same in the other places.

Reply:
Done!

Reply:
Thanks! :)